mlir/tools/mlir-tblgen/OpDocGen.cpp

```diff
@@ -195,13 +202,34 @@ static void emitOpDoc(const Operator &op, raw_ostream &os) {
     // TODO: Attributes are only documented by TableGen name, with no further
     // info. This should be improved.
```

Comment:
I guess this comment can be updated?

Reply:
I've just removed it since I don't think that's true anymore.